cli.ts

```diff
@@ -76,6 +77,25 @@ async function cli(args: ParsedArgs) {
     process.exit(0)
   }
 
+  if (args._[0] === "config" && args._[1] === "set" && args._[2] === "server") {
```

Comment:
Probably it's better to make it explicit that it's setting the fake server.
CC @seveibar 

```suggestion
  if (args._[0] === "config" && args._[1] === "set" && args._[2] === "fake-server") {
```